### PR TITLE
AG-12655 Update line series to use area series code

### PR DESCRIPTION
**Pull request by jacobp100**

### CODE REVIEW

File: packages/ag-charts-community/src/chart/series/cartesian/__image_snapshots__/line-series-test-ts-line-series-category-animation-should-animate-updating-points-to-undefined-at-0-1-snap.png
Comment (jacobp100) [inline packages/ag-charts-community/src/chart/series/cartesian/__image_snapshots__/line-series-test-ts-line-series-category-animation-should-animate-updating-points-to-undefined-at-0-1-snap.png:1]: I think this is just a change to the animation

File: packages/ag-charts-community/src/module-support.ts [hunk 1]
```diff
@@ -74,7 +74,7 @@ export * from './chart/series/seriesZIndexMap';
 export * from './chart/series/util';
 export * from './chart/series/cartesian/abstractBarSeries';
 export * from './chart/series/cartesian/cartesianSeries';
-export * from './chart/series/cartesian/lineUtil';
+export * from './chart/series/cartesian/lineUtilLegacy';
```
Comment (jacobp100) [inline packages/ag-charts-community/src/module-support.ts:77]: This is required until https://ag-grid.atlassian.net/browse/AG-13247

File: packages/ag-charts-community/src/chart/series/cartesian/lineInterpolationUtil.ts [hunk 2]
```diff
@@ -75,53 +125,55 @@ function validateAxisValuesOrder(axisValues: any[], data: SpanContext) {
 function getAxisValues(newData: SpanContext, oldData: SpanContext): AxisContext | undefined {
```
Comment (jacobp100) [inline packages/ag-charts-community/src/chart/series/cartesian/lineInterpolationUtil.ts:125]: This is fixing an integrated test (which was broken for area too)

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**